enterprise/frontend/src/embedding-sdk/components/private/SdkThemeProvider.tsx

```diff
@@ -45,5 +45,10 @@ function GlobalSdkCssVariables() {
   // the default is needed for when the sdk can't connect to the instance and get the default from there
   const font = useSelector(getFont) ?? DEFAULT_FONT;
 
-  return <Global styles={getMetabaseSdkCssVariables(theme, font)} />;
+  const styles = useMemo(
+    () => getMetabaseSdkCssVariables(theme, font),
+    [theme, font],
+  );
+
+  return <Global styles={styles} />;
```

Comment:
Memoize it so we don't need to do the style computation on every render, which gets expensive quickly.

Reply:
did you notice a slowdown?

Reply:
@WiNloSt yes, on Storybook it gets really slow, like freeze-the-whole-UI slow. I am not exactly sure if it a development-mode only slowdown though, as prod builds seems to be fine.